roles/openshift_logging_elasticsearch/templates/elasticsearch.yml.j2

```diff
@@ -6,16 +6,16 @@ script:
   indexed: on
 
 index:
-  number_of_shards: {{ es_number_of_shards | default ('1') }}
-  number_of_replicas: {{ es_number_of_replicas | default ('0') }}
+  number_of_shards: 1
```

Comment:
@ewolinetz whoops, looks like I missed that one, should be
```
number_of_shards: {{ es_number_of_shards | default ('1') }}
```
You'll also need the corresponding change for `roles/openshift_logging_elasticsearch/tasks/main.yaml` where the configmap is created - you'll need to add something like this:
```
vars:
...
       - es_number_of_shards: "{{ openshift_logging_es_number_of_shards | default(1) }}"
```